subgraph/src/utils.ts

```diff
@@ -0,0 +1,32 @@
+import { ethereum } from "@graphprotocol/graph-ts";
+import { Audit as AuditEntity, AuditInformation as AuditInfoEntity } from "../generated/schema";
+
+export function createAuditInformation(eventId: string, event: ethereum.Event): string {
+  const auditId = eventId + "-audit";
+  const audit = new AuditEntity(auditId);
+  if (!event) return auditId;
+
+  const txn = event.transaction;
+  const block = event.block;
+
+  audit.blockNumber = block.number;
+  audit.transactionHash = txn.hash;
+  audit.transactionTimestamp = block.timestamp;
+  audit.fromAddress = txn.from;
+  audit.toAddress = txn.to || txn.from;
```

Comment:
```suggestion
  audit.toAddress = txn.to;
```

There is not reason to set this value with `from`, right?

Reply:
done